pkgs/by-name/yo/yosys/package.nix

```diff
@@ -110,17 +110,31 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   enableParallelBuilding = true;
-  nativeBuildInputs = [
-    bison
-    flex
-    pkg-config
-  ];
+  nativeBuildInputs =
+    [
+      bison
+      flex
+      pkg-config
+    ]
+    ++ lib.optionals enablePython [
+      (python3.withPackages (
+        p: with p; [
+          pip
+          setuptools
+          wheel
+        ]
+      ))
+    ];
+
   propagatedBuildInputs =
     [
       libffi
       (python3.withPackages (
         pp: with pp; [
           click
+          pip
+          setuptools
+          wheel
```

Comment:
What happens without these `propagatedBuildInputs`? And are they required even when `enablePython` is `false`?

Reply:
They should probably not be added unless `enablePython` is `true`.